08-Keshia-Wijaya/wk02/5-fri/atm/js/atm.js

```diff
@@ -0,0 +1,112 @@
+// i want to get the value from checking amount so it will appear on console
+// then i want to display the value from console to #checking-balance
+// then i need to make array and change the text to number
+// next i want to create math algo to add all the value
+// i need to display the total on the checking balance
+
+const array = [];
+let sumCheckingBalance;
+let sumSavingsbalance;
+let finalTotal;
+
+
+// Checking Deposit code
+$(document).ready(function(){
+    $("#checking-deposit").click(function(){
+        let checkingAmountInput= Number(($("#checking-amount").val()));  //(get content) get the value of checking amount and store it in variabel
+        
+        array.push(checkingAmountInput); //to create an array
+        console.log(`This is a checking amount input = ${checkingAmountInput}`); //to display the checking amount on console/ dev tools
+        console.log("This is a checking amount array: ", array); // to display the value of the array that the user puts in
+        sumCheckingBalance = 0;
+        for (let i = 0; i < array.length; i++) { //to sum all of the array
+            sumCheckingBalance += array[i];
+        }
+        $("#checking-balance").text(`$${sumCheckingBalance}`); //to display the total on the checking balance
+        console.log("This is sum Deposit:", sumCheckingBalance);
+
+        if (sumCheckingBalance > 0) {
+            $(document).ready(function(){
+                $("#checking-balance").css("background-color", "white")
+            })
+        };
+    });
+});
+
+
+
+// Checking Withdraw code
+$(document).ready(function(){
```

Comment:
```suggestion
```

I recommend only having one `$(document).ready()` call. It's a bit more "DRY" and will allow your page to load slightly faster (not a big deal for a small app like this but still good to keep in mind).